Tests/FluentPostGISTests/XCTestManifests.swift

```diff
@@ -2,7 +2,7 @@ import XCTest
 
 extension GeometryTests {
     static let __allTests = [
-//        ("testGeometryCollection", testGeometryCollection),
+        ("testGeometryCollection", testGeometryCollection),
```

Comment:
Let's delete this file and rely on test discovery instead (assuming it works with the subclassing!)